/*
{
  "errCode": "SYNTAX_ERROR"
}
*/
try {
  2+1
} catch(Object o) {

} finally }